libHVCAENx527App/src/HVCAENx527.c

```diff
@@ -1237,24 +1237,7 @@ CAENx527GetAllChParVal( HVCRATE *cr, char *pname)
 #endif
 						}
 					}
-				}
-				/* this is only called from main scan loop - should we disconnect on all errors, or just use fact
-				   that CAENHV_GetSysProp() will fail and disconnect for us? */
-				else if(retval != CAENHV_OK /*retval == CAENHV_TIMEERR*/ )
-				{
-					cr->connected = 0;
-#if CAENHVWrapperVERSION / 100 == 2
-					printf("Lost connection to %s@%s: %s (%d)\n", Crate[i].name, Crate[i].IPaddr, CAENHVGetError(Crate[i].name), retval);
-					CAENHVDeinitSystem( cr->name);
-#else
-					printf("Lost connection to %s@%s: %s (%d)\n", Crate[i].name, Crate[i].IPaddr, CAENHV_GetError(Crate[i].handle), retval);
-					CAENHV_DeinitSystem( cr->handle);
-                    cr->handle = -1;
-#endif	/* CAENHVWrapperVERSION */
-					rval = 4;
-				}
-				else
-				{
```

Comment:
Should we still print an error message even if we do not disconnect from the crate?